.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2.1 # Use 2.1 to enable using orbs and other features.
+ 
+orbs: 
+  php: circleci/php@1.1.0
+
+jobs:
+  test:
+    parameters:
+      version: 
+        default: "8.0"
+        description: PHP version to install
+        type: string
+    docker:
+      - image: cimg/php:<<parameters.version>>
+    steps:
+      - checkout
+      - php/install-composer
+      - run: composer self-update
+
+      # Print php and composer versions for debugging
+      - run: php --version
+      - run: composer --version
+
+      # although not recommended, we need to update dependencies to work on php8.1
```

Comment:
Is there no way to cache/lock in the deps somehow? Bummer we need to run this step every time.

This is a very old discussion but it might help us avoid this step: https://discuss.circleci.com/t/caching-composer-dependencies/1215

Reply:
There probably is - but at the moment the total run time is only 10 seconds... so I'm not sure how much difference it would make 😅